pontoon/sync/formats/xliff.py

```diff
@@ -125,5 +127,10 @@ def save(self, locale):
 def parse(path, source_path=None, locale=None):
     with open(path) as f:
         xml = f.read().encode("utf-8")
-        xliff_file = xliff.xlifffile(xml)
+
+        try:
+            xliff_file = xliff.xlifffile(xml)
+        except etree.XMLSyntaxError as err:
+            raise ParseError(f"Failed to parse {path}: {err}")
```

Comment:
*question* What do you think about passing path and err as parameters to ParseError class and implementing a method to display a unified error message in other formats? I mean, something like this:
```
class ParseError(RuntimeError):
   def __init__(path, err):
      super().__init__(f"Failed to parse {path}: {err}")
 
ParseError(path, error)
```

Reply:
I see your point. I agree it'd be good to keep messaging in one place. But I'd rather do that separately for all custom error classes and all places where we log relevant messages.